From 56f24d40e0d26aa534214b7c856b28868db7fe2a Mon Sep 17 00:00:00 2001
From: Guoqing Jiang <guoqing.jiang@suse.com>
Date: Mon, 29 Aug 2022 18:19:06 +0800
Subject: [PATCH] Revert "Linux 5.10.139"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No


This reverts commit 665ee746071bf02ce8b7b9d729c8beab704393c2.

Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 48140575f960..234c8032c2b4 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 139
+SUBLEVEL = 138
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.34.1

